msteams-platform/sbs-gs-csharp.yml

```diff
@@ -34,61 +33,32 @@ items:
-    
-
-    ## Verify git installation
-
-    If Visual Studio installation prompts you to add `git` to the path, select it. 
-   
-    To verify the version of git installed:
-
-    1. Open a terminal window. 
-    2. Run the following command to verify the `git` version installed on your machine:
-
-        ```bash
-        $ git --version
-        ```
-
-        Output example: `git version 2.17.1.windows.2`
 
     ## Install Teams Toolkit
 
     Teams Toolkit helps simplify the development process with tools to create a project scaffolding for your app. It creates the necessary directory structure for all selected capabilities with the required files in place, ready to build the project.
 
-    Teams Toolkit is available as a Visual Studio 2019 extension.
+    Teams Toolkit is available as an extension in Visual Studio.
 
     To install Teams Toolkit extension:
 
-    1. Open Visual Studio 2019 and select the **Extensions** > **Manage Extensions**.
-    1. In the search box, enter **Teams Toolkit**.
-    1. Select **Download** next to the Teams Toolkit.
+    1. Open Visual Studio and select **Extensions** > **Manage Extensions**.
 
-        :::image type="content" source="~/assets/images/teams-toolkit-v2/install-teams-toolkit-vs19.png" alt-text="Image shows installation for Teams Toolkit"::: 
-    
-        The changes are available after you close Visual Studio 2019. It's required as Visual Studio 2019 must restart after the installing the extension.
+       :::image type="content" source="~/assets/images/teams-toolkit-v2/select-extension.png" alt-text="Select Extensions.":::
 
-    1. Select **Close**.
-    
-        :::image type="content" source="~/assets/images/teams-toolkit-v2/toolkit-installed.png" alt-text="Image shows installation done for Teams Toolkit" lightbox="~/assets/images/teams-toolkit-v2/toolkit-installed.png":::
-    
-    1. Close Visual Studio 2019.
-        
-        The VSIX installer opens with instructions for installing the extension.
+       The following Manage Extension window appears:
+
+       :::image type="content" source="~/assets/images/teams-toolkit-v2/manage-extension.png" alt-text="Manage Extensions.":::
 
-    1. Follow the instructions of VSIX installer to complete the installation of the toolkit.
-    1. Open Visual Studio 2019 again.
+    1. Select **Installed** and you can see **Teams Toolkit** extension.
     
-    The Teams Apps will be available as a template in Visual Studio 2019.
+       :::image type="content" source="~/assets/images/teams-toolkit-v2/select-ttk.png" alt-text="Select Teams Toolkit":::
 
-    You also can find the Teams Toolkit on the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=msft-vsteamstoolkit.vsteamstoolkit).
+       > [!NOTE]
+       > By default, Teams Toolkit extension is installed in Visual Studio.
```

Comment:
By default, Teams Toolkit extension is not installed in Visual Studio, you can select the Microsoft Teams Development Tools when installation.
![image](https://user-images.githubusercontent.com/25972542/183550824-fdee864f-7cb7-488c-aad0-9fd6c71497c9.png)
(You can choose to include the picture or not include it. Feel free to decide as you are a doc expert).

Reply:
Updated as mentioned.

Reply:
I didn't see the note change?

Reply:
I have removed the note.